lessons/Algebra2/exploring-quadratic-models/langs/en-us/index.adoc

```diff
@@ -0,0 +1,178 @@
+| We can build linear models that fit the decreasing half of the data...
+| We can build linear models that try to split the difference...
+
+| @image{images/fuel-efficiency-linear1.png}
+| @image{images/fuel-efficiency-linear2.png}
+| @image{images/fuel-efficiency-linear3.png}
+|===
+
+...but all of them are so far away from so many points, and our @math{R^2} is pretty bad.
+
+=== Synthesize
+
+- Why aren't linear models working for us?
+- What else could we try?
+
+== Fitting Quadratic Models @duration{35 minutes}
+
+=== Overview
+
+Students are introduced to quadratic models, and extend the sampling techniques from the linear models lesson into quadratic relationships.
+
+=== Launch
+
+Linear models are great for fitting simple relationships. But the real world isn't so simple, and, more often than not, linear models are just too primitive to handle that complexity.
+
+Relationships that rise and fall like this fuel efficiency data are called _quadratic relationships_. They form shapes that look like arches, called parabolas. Sometimes the data only includes the _rising_ part of the parabola, sometimes it only includes the _falling_ part, and and sometimes it includes both!
+
+Quadratic relationships show up all the time!
+
+- When an athlete is young, they improve as they get stronger and more skilled. But as they age, they begin to lose their speed and strength.
+- A ball dropped from a tall building will get faster and faster as it falls.
+- Many nutrients are good for you, and the more you take the healthier you are...up to a point. After that, too much of a mineral or vitamin can cause problems.
+
+[.strategy-box, cols="1", grid="none", stripes="none"]
+|===
+
+|
+@span{.title}{Going Deeper: Interaction Effects}
+
+We often think about variables of a model being _independent_ from one another, each contributing a little to the outcome we are measuring. But sometimes one input will amplify the result of another input, meaning their combined contribution is greater than the sum of their separate contributions.
+
+A quadratic relationship often means that one or more variables is _interacting_ with another variable. In our miles-per-gallon example, it turns out that a linear increase in speed has a _more than linear effect_ on mpg!
+|===
+
+[.lesson-instruction]
+Can you think of other real-world examples where a relationship is likely quadratic?
+
+You should already be familiar with the different forms of linear models:
+
+[cols="^1,^1,^1", options="header"]
+|===
+| Slope-intercept		| Point-slope				| Standard 	
+| @math{y = mx+b}		| @math{y-y_1 = m(x-x_1)}	| @math{Ax+By = C}
+|===
+
+Depending on what part of the model we care about most, we might choose to use one display over another.
+
+The same is true for quadratic models:
+
+[cols="^1,^1,^1", options="header"]
+|===
+| Standard Form 		| Vertex Form			| Factored Form
+| @math{y=ax^2+bx+c}	| @math{y=a(x−h)^2+k}	| @math{y=a(x−r_1)(x−r_2)}
+|===
+
```

Comment:
I don't think that we can expect that all students know the parts of a parabola at this point. I think we need to provide images of upward opening and downward opening parabolas here so that they can be reminded of the parts you are describing... y-intercept, minima, maxima, roots (suggest also using the terms x-intercepts and zeros here to connect to any prior knowledge they may have)

Reply:
This took some time, but it's done. Would love for you to check my Spanish translation of the new vocabulary!